server/build.gradle

```diff
@@ -29,8 +29,6 @@ BuildUtils.addLabKeyDependency(project: project, config: 'tomcatJars', depProjec
 
 dependencies
         {
-            tomcatJars "com.sun.mail:jakarta.mail:${javaMailVersion}"
-            tomcatJars "jakarta.activation:jakarta.activation-api:${activationApiVersion}" // JavaMail dependency, so it must be copied to <tomcat>/lib
```

Comment:
I assume it's on your radar to remove the tomcatJars configuration (in LabKey.groovy in the gradle plugins).

Reply:
Yes, `tomcatJars`, `tomcatLibDir`, `useEmbeddedTomcat`, et al are all going away

Reply:
For now, I'm clearing what I can from the build prior to publishing a new plugin